internal/terminal/table.go

```diff
@@ -131,6 +131,12 @@ func parseValue(value interface{}) string {
 	case nil: // leave zero-value
 	case string:
 		parsed = v
+	case bool:
+		if v {
+			parsed = "yes"
+		} else {
+			parsed = "no"
+		}
```

Comment:
was this a product decision? seems like true/false is pretty standard among developers.

Reply:
I just threw a dart here... I think initially it was showing as `TRUE`/`FALSE` and became a bit visually jarring, but we can for sure roll with `"true"`/`"false"` here if you think that's more natural  (i'll take an informal poll with product/design as well)

Reply:
I believe it's more natural and we shouldn't muddy the waters of changing a boolean to a yes/no personally.